app/models/topic_embed.rb

```
@@ -0,0 +1,82 @@
+    else
+      post = embed.post
+      # Update the topic if it changed
+      if content_sha1 != embed.content_sha1
+        revisor = PostRevisor.new(post)
+        revisor.revise!(user, absolutize_urls(url, contents), skip_validations: true, bypass_rate_limiter: true)
+        embed.update_column(:content_sha1, content_sha1)
+      end
+    end
+
+    post
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `import` is too high. [<7, 23, 6> 24.78/23]

(Metrics/AbcSize)